src/Tempest/Database/DatabaseConfig.php

```diff
@@ -8,14 +8,29 @@ final class DatabaseConfig
 {
     public function __construct(
         public readonly DatabaseDriver $driver,
-        public array $migrations = [],
+        private array $migrations = [],
```

Comment:
I guess this can be public again now that the getter and setter don't do anything? That means the getter and setter can also be removed